### PR TITLE
Fix/issue 20 Enrollment graph is properly shown

### PR DESCRIPTION
**Pull request by RidhamShah**
In this PR, I take client Timezone Offset from `/date` API and according to the Offset, the dates of the graph are generated. The Offset is also used during the query so the enrollments are shown to their respective dates irrespective of the client timezone. 

Now, during the `Last 7 Days` option instead of day, the X-Axis will show the date as `mm/dd` and on hovering the graph it will show the full date with the day as `mm/dd/yyyy - day`.
image

### CODE REVIEW

**Review by VivekFitkariwala — APPROVED** [review]

File: frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class EnrollmentOverTimeComponent implements OnChanges, OnInit, OnDestroy
   graphInfoSub: Subscription;
   isGraphLoading$ = this.experimentService.isGraphLoading$;
 
-  constructor(private experimentService: ExperimentService) { }
+  constructor(private experimentService: ExperimentService) { this.formateXAxisLabel = this.formateXAxisLabel.bind(this) }
```
Comment (JD2455) [inline frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts:52]: @RidhamShah Can you please add semicolon in the end of statement inside the constructor?

File: frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts [hunk 2]
```diff
@@ -79,12 +79,14 @@ export class EnrollmentOverTimeComponent implements OnChanges, OnInit, OnDestroy
...
     });
     // Used to fetch last 7 days graph data
-    this.experimentService.setGraphRange(this.selectedDateFilter, this.experiment.id);
+    this.experimentService.setGraphRange(this.selectedDateFilter, this.experiment.id, -new Date().getTimezoneOffset());
   }
 
   // remove empty series data labels
   formateXAxisLabel(value) {
-    return !isNaN(value) ? '' : value.substring(0, 3);
+    if(this.selectedDateFilter === DATE_RANGE.LAST_SEVEN_DAYS)
```
Comment (JD2455) [inline frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts:87]: @RidhamShah Can you please format this part of code to add curly braces for more readability?

File: frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts (hunk 1 above)
Comment (RidhamShah) [reply frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts]: code updated

File: frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts (hunk 2 above)
Comment (RidhamShah) [reply frontend/projects/abtesting/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts]: code updated

**Review by msandbothe — APPROVED** [review]

**Pull request merged**